### ISSUE TITLE
the influence of the patch size

### ISSUE THREAD

**Issue by duanwenhong** [zhxl0903/CSCD94_VESPCN#7]
hi,thanks for your work.I have some questions about the patch size.I use 32 patch size and i noticed that i can not learn the optical-flow.Can you share your insights .Thanks

**Comment by zhxl0903** [comment]
Hi,
Thank you for your question. After some testing, I determined the billinear sampler for MC is faulty.
I have replaced the old billinear sampler with a working one from https://github.com/iwyoo/tf-bilinear_sampler . Mode 0 will now output the correctly warped frame along with the optical flow maps.
zhxl0903

**Comment by duanwenhong** [comment]
Hi,thank you for your reply. I have another question about the weights.  I have noticed that you modify the weights from 0.01 to 1e-6.  When I test with this weights(1e-6) but I donot replace the bilinear sampler, it also has a better effect. So I have this question why you modify the weights.  In addition, I have another question why you use tf.nn.tanh after the t1_fine_warp. Thanks for your explanation.

**Comment by zhxl0903** [comment]
Hi,

It looks like the weights are set too high which over penalised the huber loss, reducing the flow map gradients to zero. Hence, the flows become near constant 0. I have replaced reduce_sum in huber loss with reduce mean. Mode 0 now generates good flowmaps with a weight around 1e-3. tf.nn.tanh will work with normalised images in the range (-1, 1) or (0, 1), if we have MC loss term.      

zhxl0903

**Comment by duanwenhong** [comment]
Hi,
thanks for your reply. I gained good results for flowmaps with your weights. Thank you for your kindness help.

**Comment by duanwenhong** [comment]
Hi, I have another question about the billinear sampler function that why you donot set the value of normalize parameter True. I think if we donot set, it is almost the same as original image. Thanks for your comments.

**Comment by duanwenhong** [comment]
Hi,thanks for your work.
I have another question about the billinear sampler function that why you donot set the value of normalize parameter True.Thanks for your explanation.
